stream_alert/athena_partition_refresh/main.py

```diff
@@ -257,11 +257,9 @@ def check_table_exists(self, table_name):
         if query_success and query_resp['ResultSet']['Rows']:
             return True
 
-        LOGGER.info('The streamalert table \'%s\' does not exist. '
-                    'For alert buckets, create it with the following command: \n'
-                    '$ python manage.py athena create-table '
-                    '--type alerts --bucket s3.bucket.id',
-                    table_name)
+        LOGGER.info('The streamalert table \'%s\' does not exist.', table_name)
+        LOGGER.info('Create it with the following command: '
```

Comment:
The following command is to show `create-table` helper. The statement is misleading here.

Reply:
yep! I did that on purpose, so users could figure out the settings they need

Reply:
Since the command you point to doesn't actually _create_ the table, like the log statement indicates, maybe change the statement to something like `'For help with creating tables, use the following command...'`